images/description/presentation-large.svg

```diff
@@ -0,0 +1,8 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <style>
+    .st0{fill:#74777B;}
```

Comment:
We need to specify it on the SVG file itself, but when this gets consumed in the `iron-` stuff, we can exclude it.